File: Modul-3/Soal 3/tempCodeRunnerFile.c
#include <stdio.h>

int main(){
    int angka;
    printf("Masukkan Bilangan Bulat: ");
    scanf("%d", &angka);

    if(angka > 0){printf("Positif");}
    else if(angka < 0){printf("Negatif");}
    else if (angka==0){printf("Nol");}
}

File: Modul-4/Soal 1/tempCodeRunnerFile.py
i = 1
j = int(input("Masukkan Angka Kelipatan: "))
k = input("Masukkan Simbol: ")

while i <= 50:
    if i % j == 0:
        print(k, end=" ")
    else:
        print(i, end=" ")
    i += 1